Package.swift

```diff
@@ -0,0 +1,31 @@
+// swift-tools-version:5.1
```

Comment:
Might as well make this version 5.3 now?
```suggestion
// swift-tools-version:5.3
```

Reply:
The general recommendation is to use the minimum tool version required for processing the manifest. Since none of the shiny 5.3 features are required for Mantle at the moment (there are namely no embedded resources or binary dependencies involved), I set the version to 5.1 so that earlier versions of Xcode are supported.

Of course if you think it is best to support only the last version of Xcode (which I do for projects I am working on, so I totally understand), bumping the tool version to 5.3 is the right approach.

Reply:
Makes total sense. Arguably someone using SPM to include Mantle is likely to use a newer version of Xcode but that's a wild guess. Please disregard the suggestion then.